person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.ran(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  private :of_age?
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || parent_permission == true
+  end
```

Comment:
- [ ] This method does not work as expected. When I declare a Person 17 years of age I get an error (see the image). This is because the method is not using the defined `@parent_permission` instance variable. Kindly fix this.

  **SUGGESTION:** You can define your method to look like so:
  ```
  def can_use_services?
    of_age? || @parent_permission
  end
  ```

  ![tumwo-def](https://user-images.githubusercontent.com/36542986/156176765-4108f570-8eb6-4a02-890f-74c8c521a1f8.jpg)